src/lib/pickles/scalar_challenge.ml

```diff
@@ -157,6 +157,13 @@ let test (type f)
   let open Impl in
   let module T = Internal_Basic in
   let n = 128 in
+  let module Field_constant = struct
+    include Field.Constant
+
+    type nonrec bool = bool
```

Comment:
I have the feeling I have read this diff before, not sure if we could reuse the definition existing in another file (cannot remember which)